src/main/java/frc/robot/subsystems/intake/roller/IntakeStates.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.intake.roller;
+
+public enum IntakeStates {
+
+	NOTE_TO_SHOOTER(0),
+	INTAKE(0),
+	STOP(0),
+	ELEVATOR_TO_SHOOTER(0);
```

Comment:
make `STOP` state last
```suggestion
	ELEVATOR_TO_SHOOTER(0),
	STOP(0);
```